### DISCUSSION TITLE
Does Flex component has `gap` prop?

### DISCUSSION THREAD

**Discussion by hurifajri** [chakra-ui/chakra-ui#4169] [Q&A]
I am seeing that only Grid component which has `gap` prop. Why Flex component doesn't?

**Comment by with-heart** [comment]
Hi @hurifajri! I re-opened #2720 to add support for `flex-gap`. 

We didn't have support for it due to lack of support on Safari but they added support at the end of April.

**Reply by hurifajri** [reply]
Hi @with-heart 
Thanks for your response. Looking forward for it :)
